### DISCUSSION TITLE
使用命令行用法添加第二个账号问题

### DISCUSSION THREAD

**Discussion by david4423** [messense/aliyundrive-webdav#462] [Q&A] [answered]
### 问题描述

我在openwrt的TTYD 终端这里输入以下命令后可以访问webdav文件，但是只要一退出终端就无法访问了，请问怎么保存第二个账号设置？

### 重现步骤

aliyundrive-webdav --auto-index   --no-trash --cache-size 1000 --cache-ttl 600 --debug --host 192.168.1.1  --port 8080 --refresh-token xxxxxxxxxxxxx--read-buffer-size 10485760

### 版本

aliyundrive-webdav 1.5.0

### 运行平台

openwrt

### 日志

没

**Comment by messense** [comment, marked as answer]
写个 service 文件，参考 https://github.com/messense/aliyundrive-webdav/blob/main/openwrt/aliyundrive-webdav/files/aliyundrive-webdav.init 和 https://blog.csdn.net/letterwhite/article/details/26026425

**Reply by david4423** [reply]
我路由器地址是192.168.8.1，     local host=$(uci_get_by_type server host 192.168.8.1)service 文件里代码格式是这样写吗？我这样试了下，没有生效，重启路由后又导致mosdns启动不了了，太复杂了对于小白
